WordPress/gutenberg
# Comments block: Change ID to `core/comments`

Left as a draft to have some feedback about the rename first, the options we have are:

- Leave it as is ( is not a Query Loop yet)
- Rename it (inconsistency between ID - class,  and the rest of the name on the interface, folders, etc). We cannot update the block name, as has been stable a couple of months.

## What?

Renaming `Comments Query Loop` with `Comments`

## Why?

Right now, Comments Query Loop is not working like a Query block, it has not any option and is just rendering the comments, being able to add styling to them.

## How?

Search and replace!

## Testing Instructions

- On `trunk`, edit the "Single" template in the Site Editor to add the "Comments" block.
- Change some of the block's properties, e.g. text color or font size.
- Switch to the Code Editor and verify that the block's ID is `wp:comments-query-loop`. Copy-paste the block attributes, and the outermost `<div />` tag (including its HTML attributes) to a text editor of your choice; we'll need them later.
- Switch back to the Visual Editor.
- Switch to this branch (`update/rename-comments-query-loop`).
- Reload the template in the Site Editor.
- Open your browser console, and verify that it contains an "Updated blocks: core/comments` message.
- Switch to the Code Editor. Verify that the block ID is now `wp:comments` there ✅  Compare the block attributes and the outermost `<div />` tag to the ones you copied earlier. They should be identical, except now there should be an additional `wp-block-comments-query-loop` class (for backwards compatibility).
- Switch back to the Visual Editor.
- Verify that the block still works as before.
- Remove the block, and insert a new "Comments" block. Verify in the browser console that it doesn't trigger a block deprecation.
- Switch to the Code Editor, and verify that it _doesn't_ have the `wp-block-comments-query-loop` class.

Finally, try the same in the Post Editor.

## Screenshots or screencast
